roles/ovirt-repositories/tasks/backup-repos.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: creating directory repo-backup for repo files
+  file:
+    path: /tmp/repo-backup
+    state: directory
+
+- name: copy content of yum.repos.d to backup folder
+  shell: 'cp /etc/yum.repos.d/*.repo /tmp/repo-backup'
```

Comment:
Hmm, any timestamp? Maybe do the path configurabale?

Reply:
ok